16/default.conf

```diff
@@ -0,0 +1,250 @@
+        proxy_set_header Host $host;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_cache_bypass $http_upgrade;
+
+        location ~* /system/files/ {
+            include fastcgi_params;
+            fastcgi_param QUERY_STRING q=$uri&$args;
+            fastcgi_param SCRIPT_NAME /index.php;
+            fastcgi_param SCRIPT_FILENAME $document_root/index.php;
+            fastcgi_hide_header 'X-Drupal-Cache';
+            fastcgi_hide_header 'X-Generator';
+            fastcgi_pass upstream;
+            log_not_found off;
+        }
+
+        location ~* /sites/.*/files/private/ {
+            internal;
+        }
+
+        location ~* /files/styles/ {
+            access_log off;
+            expires 30d;
+            try_files $uri @drupal;
+        }
+
+        location ~* /sites/.+/files/.+\.txt {
+            access_log off;
+            expires 7d;
+            tcp_nodelay off;
+            open_file_cache max=1000 inactive=30s;
+            open_file_cache_valid 30s;
+            open_file_cache_min_uses 2;
+            open_file_cache_errors off;
+        }
+
+        location ~* /sites/.*/files/advagg_css/ {
+            expires max;
+            add_header ETag '';
+            add_header Last-Modified 'Wed, 20 Jan 1988 04:20:42 GMT';
+            add_header Accept-Ranges '';
+            location ~* /sites/.*/files/advagg_css/css[_[:alnum:]]+\.css$ {
+                access_log off;
+                try_files $uri @drupal;
+            }
+        }
+
+        location ~* /sites/.*/files/advagg_js/ {
+            expires max;
+            add_header ETag '';
+            add_header Last-Modified 'Wed, 20 Jan 1988 04:20:42 GMT';
+            add_header Accept-Ranges '';
+            location ~* /sites/.*/files/advagg_js/js[_[:alnum:]]+\.js$ {
+                access_log off;
+                try_files $uri @drupal;
+            }
+        }
+
+        location ~* /admin/reports/hacked/.+/diff/ {
+            try_files $uri @drupal;
+        }
+
+        location ~* ^.+\.(?:cur|jpe?g|gif|htc|ico|png|xml|otf|ttf|eot|woff|woff2|svg)$ {
+            access_log off;
+            expires 30d;
+            tcp_nodelay off;
+            open_file_cache max=3000 inactive=120s;
+            open_file_cache_valid 45s;
+            open_file_cache_min_uses 2;
+            open_file_cache_errors off;
+        }
+
+        location ~* ^.+\.(?:css|js)$ {
+            access_log off;
+            expires 30d;
+            tcp_nodelay off;
+            open_file_cache off;
+        }
+
+        location ~* ^.+\.(?:pdf|pptx?)$ {
+            expires 30d;
+            tcp_nodelay off;
+        }
+
+        location ~* ^(?:.+\.(?:htaccess|make|txt|engine|inc|info|install|module|profile|po|pot|sh|.*sql|test|theme|tpl(?:\.php)?|xtmpl)|code-style\.pl|/Entries.*|/Repository|/Root|/Tag|/Template)$ {
```

Comment:
The `test` should be removed here as this image targeted to d8
Old images also needs update - otherwise no way to use "test" in URL